ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -235,3 +235,18 @@ Get Instance Type Of Node
     [Arguments]    ${node}
     ${type} =    Run    oc get Node -o json ${node} | jq '.metadata.labels["beta.kubernetes.io/instance-type"]'
     RETURN    ${type}
+
+Container Image Url Using Image Digest Instead Of Tags Based On Project Name
```

Comment:
I'd probably rename it to sth like `Container Image Url Shuld Use [...]`

Reply:
Done